tools/mtouch/mtouch.csproj

```diff
@@ -202,9 +202,6 @@
     <Compile Include="..\linker\ObjCExtensions.cs">
       <Link>Xamarin.Linker\ObjCExtensions.cs</Link>
     </Compile>
-    <Compile Include="..\linker\RemoveSelectors.cs">
```

Comment:
Is it only in XM/Classic? Then it might make sense to not add the step in XM/Unified in mmp (which is probably out-of-scope for this PR).